converter/webtool_pdf2csv.py

```
@@ -0,0 +1,133 @@
+from dotenv import load_dotenv, find_dotenv
+import streamlit as st
+from pdf2image import convert_from_path
+from openai import OpenAI
+import cv2
```

Ruff (0.11.9)

`cv2` imported but unused

Remove unused import: `cv2`

(F401)